packages/fx-core/src/plugins/resource/frontend/ops/scaffold.ts

```diff
@@ -94,7 +96,15 @@ export class FrontendScaffold {
       );
       return await FrontendScaffold.fetchZipFromUrl(templateUrl);
     } catch (e) {
+      Logger.debug(e.toString());
```

Comment:
Is this logging only for debug and we need to remove it from PROD?

Reply:
As we synced offline, remain this.